config/packages/views.yaml

```
@@ -1,6 +1,6 @@
 ezpublish:
     system:
-        site_group:
+        site:
             pagelayout: "pagelayout.html.twig"
```

```suggestion
            pagelayout: pagelayout.html.twig
```

Optionally we can drop `"`s